src/main/java/com/iexec/resultproxy/version/VersionService.java

```diff
@@ -2,21 +2,22 @@
 
 import com.iexec.common.utils.VersionUtils;
 
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
 import org.springframework.stereotype.Service;
 
 @Service
 public class VersionService {
 
-    @Value("${version}")
-    private String version;
+    @Autowired
+    BuildProperties buildProperties;
```

Comment:
private field + injection in constructor?

Reply:
see https://github.com/iExecBlockchainComputing/iexec-result-proxy/pull/45/commits/2118178c58a92d13b24f8a4c917f3d078dc375d7